ansible/inventory/teleop/group_vars/client.yml

```diff
@@ -3,7 +3,7 @@ image: "shadowrobot/shadow-teleop" # Docker image name
 tag: "kinetic-v0.0.11" # Tag of the exact Docker hub image to pull
 reinstall: false # Flag to know if the docker container should be fully reinstalled
 container_name: "teleop" # Name of the docker container
-nvidia_docker: 0 # Which version of nvidia-docker is enabled (0 means it is disabled)
+nvidia_docker: 2 # Which version of nvidia-docker is enabled (0 means it is disabled)
```

Comment:
please change it to 1 by default

Reply:
I think 1 is not tested yet

Reply:
Changed it to 1, will test with real hardware